cmd/kata-manager/kata-manager.sh

```diff
@@ -0,0 +1,258 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+[ -n "$DEBUG" ] && set -x
+
+typeset -r script_name=${0##*/}
+
+typeset -r doc_repo="github.com/kata-containers/documentation"
+typeset -r test_repo="github.com/kata-containers/tests"
+
+typeset -r default_config_file="/usr/share/defaults/kata-containers/configuration.toml"
+typeset -r agent_debug="agent.log=debug"
+
+verbose="no"
+config_file=
+
+usage()
+{
+	cat <<EOT
+Usage: ${script_name} [options] [command]
+
+Description: Install and configure Kata Containers.
+
+Options:
+
+  -c <file> : Specify full path to configuration file
+              (default: '$default_config_file').
+  -h        : Display this help.
+  -v        : Verbose output.
+
+Commands:
+
+  enable-debug   : Enable full debug for all system components.
+  install        : Install the packaged version of Kata Containers.
```

Comment:
this should be `install-packages` as `install` fails:
```
[fuentess@centos-kata-manager kata-manager]$ ./kata-manager.sh install
ERROR: invalid command: 'install'
```

Reply:
Fixed.